image_provisioner/playbooks/roles/docker-config/tasks/main.yaml

```diff
@@ -38,22 +48,26 @@
      ignore_errors: true
  
    - name: waiting for server to come back
-     local_action: wait_for host={{ inventory_hostname }} state=started delay=15 timeout=300
+     local_action: wait_for
+     args:
+       host: "{{ inventory_hostname }}"
+       port: 22
+       state: started
+       delay: 15
+       timeout: 300
```

Comment:
Nope, just kept the same value as before however we were previously just waiting the fixed timeout. Will double timeout.